drivers/wifi/infineon/airoc_wifi.c

```diff
@@ -363,6 +363,37 @@ static void airoc_wifi_network_process_ethernet_data(whd_interface_t interface,
 	}
 }
 
+static enum ethernet_hw_caps airoc_get_capabilities(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+	return ETHERNET_HW_FILTERING;
```

Comment:
It would make sense to return all supported capabilities here (in different commit), like supporting 100Mbit/sec speed etc.

Reply:
This is Wi-Fi driver. All other features (e.g. 100Mbps) are related to Ethernet and not supported in the case of Wi-Fi.

Reply:
Ok, fair enough.